src/complete-data/state-array.js

```diff
@@ -0,0 +1,54 @@
+export const stateArray = {
```

Comment:
Hell yeah for using an object to store all the state abbreviations. One nitpicky detail, I would rename the const to something other than stateArray, since it's an object. Maybe stateAbrv or something?